Qpf/Macro/Comp.lean

```diff
@@ -71,13 +79,14 @@ where
     let F : Q(CurriedTypeFun.{u,u} $depth) := F
 
     trace[QPF] "F := {F}\nargs := {args.toList}\ndepth := {depth}"
-    QQ.check F
+    check F
```

Comment:
```suggestion
    F.check
```

The function `QQ.check` was moved to `QQ.Quoted.check`. Your suggestion would use `Lean.Meta.check`, which is slightly different: the `Lean` function checks that the expression is well-typed (for some type), whereas the `Qq` version additionally checks that the type corresponds to the declared type (i.e., `CurriedTypeFun.{u, u} $depth`)

Reply:
I wasn't aware of that, thanks!